articles/migrations/guides/extensibility-node8.md

```diff
@@ -1,57 +1,67 @@
 ---
-title: Migration Guide - Extensibility and Node.js v8
+title: Migration Guide - Extensibility and Node 8
 description: This article covers the Auth0 features/modules affected, as well as our recommendations to ensure a smooth migration process.
 toc: true
 ---
-# Migration Guide: Extensibility and Node.js v8
+# Migration Guide: Extensibility and Node 8
 
 :::warning
-The Node.js 8 runtime will be available to all public cloud customers on **April 17th, 2018**.
+The Node 8 runtime will become available to all public cloud customers on **April 17, 2018**.
 :::
 
-Beginning 2018 April 30, [Node.js v4 will be going out of long-term support (LTS)](https://github.com/nodejs/Release#release-schedule), which means that the Node.js development team will no longer be back-porting critical security fixes to this version.
+Beginning April 30, 2018, [Node.js v4 will be going out of long-term support (LTS)](https://github.com/nodejs/Release#release-schedule), which means that the Node.js development team will no longer be back-porting critical security fixes to this version and this _could_ expose your extensibility code to security vulnerabilities.
 
-As such, Auth0 will be migrating from Node.js v4 to Node.js v8.
+As such, Auth0 will be migrating from Node 4 to Node 8.
+
+:::note
```

Comment:
I'd remove the `note` element and leave the text as is. The `note` is supposed to be used for extra information and we have been asked from design to use it less (when too many things call for attention the end result is the opposite).